Automattic/sensei
# Add explanation about the "Display Course Author" setting

## Proposed Changes

* We have a setting to display the author on the Course Archive and My Courses pages. But it only works for the legacy pages (with shortcodes or automatically injected in the template). So I added an extra comment that if they are using blocks, they should remove the author there.

Reported in p1708018654910029-slack-C02P7FHLVR9

## Screenshots

<img width="1071" alt="Screenshot 2024-02-26 at 20 32 27" src="https://github.com/Automattic/sensei/assets/876340/fe29fe7c-9ddb-4f2b-a5f7-9b69606cfb56">


## Testing Instructions


1. Open the Sensei settings in the course tabs.
2. Check that the text is updated and it looks good enough. cc @donnapep to see if it looks good enough. =)

## Pre-Merge Checklist
